.github/workflows/main.yml

```diff
@@ -0,0 +1,29 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: macos-latest
+
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
```

Comment:
Remove this comment (or replace it with a `name` for the step).

Reply:
removed in bb84fb0cafeb27191c29c340d94b300b68fe9192